src/components/common/ColonyActions/ActionDetailsPage/staticMaps.ts

```diff
@@ -97,6 +97,7 @@ export const ACTIONS_EVENTS: ActionsEventsMap = {
   [ColonyActionType.EmitDomainReputationReward]: [
     ColonyAndExtensionsEvents.ArbitraryReputationUpdate,
   ],
+  [ColonyActionType.MakeArbitraryTransaction]: SAFE_EVENTS,
```

Comment:
I think this is wrong as only one of these events should be shown depending on the type of safe transaction that was carried out.

Probably best to remove this line as #990 handles this integration.

Unless I am missing something and if so sing out :bird:

Reply:
As far as I understand those mappings, they only indicate the type of events that _could_ be displayed on the action page. So we are not necessarily telling it that all those events will appear at the same time. For example, `MOTION_EVENTS` has these:

```
const MOTION_EVENTS = [
  ColonyAndExtensionsEvents.MotionCreated,
  ColonyAndExtensionsEvents.MotionStaked,
  ColonyAndExtensionsEvents.ObjectionRaised,
  ColonyAndExtensionsEvents.MotionFinalized,
  ColonyAndExtensionsEvents.MotionRewardClaimed,
];
```

The only event in this array that is 100% guaranteed to appear on the motion page is `MotionCreated`. The rest _could_ appear depending on certain conditions (This also happens with the recovery mode events). I treat `MakeArbitraryTransaction` the same way as you can see :)

Reply:
perhaps I don't understand this properly then, lets have a quick chat but my understanding is that each event will be mapped over and displayed in `ActionDetailsPageFeed.tsx` with:
```tsx
  const events =
    JSON.parse(actionData.individualEvents as string) ||
    arbitraryTransactionEventParser(actionData) ||
    ACTIONS_EVENTS[actionData.type];

  return (
    <>
      {events?.map((event) => (
        <ActionsPageEvent
          actionData={actionData}
          eventName={event?.type || event}
          key={event?.id || event}
          eventId={event?.id || event}
        />
      ))}
    </>
  );
```

I experimented with setting up `SAFE_EVENTS` like you have here and could not see a way for them to appear only with certain conditions.

Reply:
Sorry, I will try to explain myself better.

What I said was in response to this comment specifically:
> I think this is wrong as only one of these events should be shown depending on the type of safe transaction that was carried out.

It's my understanding (I could be wrong) that when we group events together as we do with `SAFE_EVENTS` or `MOTION_EVENTS`, we are _not_ telling `ActionDetailsPageFeed` that _every single_ event inside those objects should appear, instead, we are telling it "Hey, for this type of action page, every event inside this object is an event that COULD BE displayed."